assets/apps/components/src/Controls/FormTokenFieldControl.js

```diff
@@ -26,21 +43,34 @@ const FormTokenFieldControl = ({
 
 	const getValueKeys = () => {
 		// Convert option ids into labels
-		const valueKeys = [];
-		for (const optionId of value) {
-			const wantedPost = choices.find((option) => {
-				return option.id === optionId;
-			});
-			if (wantedPost === undefined || !wantedPost) {
-				continue;
+
+		// Keep compatibility with previous version
+		if (
+			suggestions.length !== 0 &&
+			value.length > 0 &&
+			typeof value[0] !== 'object'
```

Comment:
`null` and `[]` are considered objects when evaluated with `typeof`. I'm guessing that won't be the case, but it's good to be aware of that. 

I'd rather base the check on `===` instead if `value[0]` was previously a string or a more concrete primitive type.

Reply:
The FormTokenField accepts as suggestions only a simple array ( eg. an array of ids, an array of slugs, an array of titles, etc. ).
For the suggestions, I must pass to the control the labels that will appear on the front end. When a user selects a new value, I must correlate the selected label with an ID. Since the title is not unique ( two posts can have the same title), I choose the slug instead. So for REST, I would actually need "slugs that start with".